src/dsp_tools/commands/project/create/project_create_all.py

```diff
@@ -359,17 +355,60 @@ def _create_users(
         )
         try:
             user_local.create()
+            print(f"    Created user '{username}'.")
+            logger.info(f"Created user '{username}'.")
         except BaseError:
             print(f"    WARNING: Unable to create user '{username}'.")
             logger.exception(f"Unable to create user '{username}'.")
             overall_success = False
-            continue
-        print(f"    Created user '{username}'.")
-        logger.info(f"Created user '{username}'.")
 
     return overall_success
 
 
+def _add_user_to_groups_and_project(
```

Comment:
The code that I wrote here is a hybrid: It is neither in the legacy-monster-class `User`, nor is it in a proper client (because the clients will only be introduced by the big refactoring/rewriting PR).

The reason I did this: RDU told that this feature would be very useful for them. So I want to ship it soon, and not wait for the big rewriting PR.

Reply:
fair enough